lms/migrations/versions/e5a9845d55d7_clean_grouping_and_assignment_titles.py

```diff
@@ -0,0 +1,49 @@
+"""Clean grouping and assignment titles.
+
+Revision ID: e5a9845d55d7
+Revises: f8ff7e49cbbf
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "e5a9845d55d7"
+down_revision = "f8ff7e49cbbf"
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+
+    result = conn.execute(
+        sa.text(
+            """
+        UPDATE assignment SET title = TRIM(title)
+        WHERE TRIM(title) != title;
```

Comment:
Does this really make a difference? Couldn't we just get away with `UPDATE assignment SET title = TRIM(title)`?

Reply:
Yes, while the where makes the query slower it should lock less rows which should be safer

Reply:
Ah, good point.